grammar.js

```diff
@@ -119,6 +119,15 @@ function statements(trailing) {
 	let semicolon     = trailing ? [] : [';'];
 	
 	return Object.fromEntries([
+		[rn('var'),         $ => seq(
+			$.kVar,
+			$.identifier,
+			optional(seq(
+				':',
+				field('type', $.typeref)
+			)),
+			optional(seq($.kAssign, $._expr)
+		))],
```

Comment:
Is this part really needed? The node doesn't seem to be used anywhere in the tests.